content/shmem_test_some_vector.tex

```diff
@@ -0,0 +1,85 @@
+\apisummary{
+    Indicate whether at least one variable within an array of variables on the local \ac{PE} meets \oldtext{a} \newtext{its} specified test condition.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+size_t @\FuncDecl{shmem\_test\_some\newtext{\_vector}}@(TYPE *ivars, size_t nelems, size_t *indices, const int *status,
+    int cmp, TYPE *cmp_values);
+\end{C11synopsis}
+where \TYPE{} is one of the point-to-point synchronization types specified by
+Table \ref{p2psynctypes}.
+
+\begin{Csynopsis}
+size_t @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_test\_some\newtext{\_vector}}@(TYPE *ivars, size_t nelems, size_t *indices,
+    const int *status, int cmp, TYPE *cmp_values);
+\end{Csynopsis}
+where \TYPE{} is one of the point-to-point synchronization types and has a
+corresponding \TYPENAME{} specified by Table \ref{p2psynctypes}.
+
+\begin{apiarguments}
+
+  \apiargument{IN}{ivars}{A pointer to an array of remotely accessible data
+    objects.}
+  \apiargument{IN}{nelems}{The number of elements in the \VAR{ivars} array.}
+  \apiargument{OUT}{indices}{An array of indices of length at least
+    \VAR{nelems} into \VAR{ivars} that satisfied the test condition.}
+  \apiargument{IN}{status}{An optional mask array of length \VAR{nelems}
+    that indicates which elements in \VAR{ivars} are excluded from the test set.}
+  \apiargument{IN}{cmp}{A comparison operator from Table~\ref{p2p-consts} that
+    compares elements of \VAR{ivars} with \newtext{elements of}
+    \VAR{cmp\_value\newtext{s}}.}
+  \newtext{\apiargument{IN}{cmp\_values}{An array of length \VAR{nelems}
+    containing values to be compared with the respective objects in \VAR{ivars}.}}
+
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_test\_some\newtext{\_vector}} routine indicates whether at
+    least one entry in the test set specified by \VAR{ivars} and \VAR{status}
+    satisfies the test condition at the calling \ac{PE}.  This routine does not
+    block and returns zero if no entries in \VAR{ivars} satisfied the test
+    condition.  \newtext{This routine compares each element of the \VAR{ivars}
+    array in the test set with each respective value in \VAR{cmp\_values}
+    according to the comparison operator \VAR{cmp} at the calling \ac{PE}}.
+    This routine tests all elements of \VAR{ivars} in the test set at least
+    once, and the order in which the elements are tested is unspecified.  If an
+    entry $i$ in \VAR{ivars} within the test set satisfies the test condition,
+    a series of calls to \FUNC{shmem\_test\_some\newtext{\_vector}} must
+    eventually return $i$.
```

Comment:
I think the closing sentence of this paragraph is a mistaken copy-paste from elsewhere. The closing sentence of the following paragraph seems more appropriate for this particular function.